app/_how-tos/private-hosted-zones.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Configure private hosted zones for Dedicated Cloud Gateway
+description: 'Enable private DNS resolution for your Dedicated Cloud Gateway using either a private hosted zone or an Outbound DNS Resolver.'
+content_type: how_to
+permalink: /dedicated-cloud-gateways/private-hosted-zones/
+breadcrumbs:
+  - /dedicated-cloud-gateways/
+products:
+    - gateway
+
+works_on:
+    - konnect
+
+tldr:
+  q: How do I configure a Private Hosted Zone for my Dedicated Cloud Gateway?
+  a: Use the AWS CLI to authorize VPC association, then call the {{site.konnect_short_name}} API to attach the hosted zone for private DNS resolution.
+related_resources:
+  - text: Dedicated Cloud Gateways
+    url: /dedicated-cloud-gateways/
+  - text: Outbound DNS resolver
+    url: /dedicated-cloud-gateways/outbound-dns-resolver/
+  - text: Amazon VPC Documentation
+    url: https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html
+prereqs:
+  skip_product: true
+  inline:
+  - title: "Dedicated Cloud Gateway"
+    include_content: prereqs/dedicated-cloud-gateways
+  - title: "AWS CLI"
+    include_content: prereqs/aws-cli
+  - title: "AWS private hosted zone"
+    content: |
+      This tutorial requires:
+      - An AWS subscription with access to [private hosted zones](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/hosted-zone-private-creating.html)
+      - Permission to run `route53:create-vpc-association-authorization`
+      - A VPC in your AWS account to associate with the zone
+
+      You'll also need the following information:
+      - Your `hosted-zone-id`
+      - Your `VPCRegion`
+      - Your `VPCId`
+
+      Create environment variables to store these credentials:
+
+      ```sh
+      export AWS_HOSTED_ZONE_ID='YOUR_HOSTED_ZONE_ID'
+      export AWS_VPC_REGION='YOUR_VPC_REGION'
+      export AWS_VPC_ID='YOUR_VPC_ID'
+      ```
+
+---
+
+
+## Associate {{site.konnect_short_name}} with your private hosted zone
+
+Using the AWS CLI, create an associate between the hosted zone and the VPC:
+
+```sh
+aws route53 create-vpc-association-authorization \
+  --hosted-zone-id Z082811935OXJB57VZOSV \
+  --vpc VPCRegion=us-east-2,VPCId=$AWS_VPC_ID
+```
```

Comment:
I got the following error for this:
```
An error occurred (InvalidClientTokenId) when calling the CreateVPCAssociationAuthorization operation: The security token included in the request is invalid.
```
I did go through the AWS CLI setup and added the access keys + the same region as we're using for the VPC.

Reply:
The Konnect configuration and validation steps seemed to work so 🤷‍♀️